lib/ansible/modules/system/user.py

```diff
@@ -2509,6 +2566,9 @@ def main():
             expires=dict(type='float'),
             password_lock=dict(type='bool'),
             local=dict(type='bool'),
+            profile=dict(type='str', default=''),
```

Comment:
Setting the default to an empty string here then testing for `None` results in this these options always being appended to the end of the command because `'' != None`.

Omit the `default` in order to have it be `None` when no value is provided.

Reply:
Oh, okay, will change it in the next ccommit